Makefile.js

```diff
@@ -112,21 +94,21 @@ target.browserify = function() {
     // 1. create temp and build directory
     if (!test("-d", TEMP_DIR)) {
         mkdir(TEMP_DIR);
-        mkdir(TEMP_DIR + "/lib");
+        mkdir(`${TEMP_DIR}/lib`);
```

Comment:
Looks like the `/` is unnecessary here. I wonder if we should just use `path.join()`?